src/spikeinterface/core/waveforms_extractor_backwards_compatibility.py

```diff
@@ -248,6 +248,9 @@ def format(self):
     def has_extension(self, extension_name: str) -> bool:
         return self.sorting_analyzer.has_extension(extension_name)
 
+    def load_extension(self, extension_name: str):
+        return self.sorting_analyzer.load_extension(extension_name)
```

Comment:
```suggestion
        return self.sorting_analyzer.get_extension(extension_name)
```
All extensions are pre-loaded